.circleci/config.yml

```diff
@@ -3,10 +3,18 @@ version: 2
 jobs:
     "node":
         docker:
-            - image: circleci/node:11.10.1
+            - image: circleci/python:3.7-stretch-node
 
         steps:
             - checkout
+            - run:
+                name: "Update Node.js and npm"
+                command: |
+                  curl -sSL "https://nodejs.org/dist/v11.10.1/node-v11.10.1-linux-x64.tar.xz" | sudo tar --strip-components=2 -xJ -C /usr/local/bin/ node-v11.10.1-linux-x64/bin/node
+                  curl https://www.npmjs.com/install.sh | sudo bash
+            - run:
+                name: Check current version of node
+                command: node -v
```

Comment:
What does this command do? Is it just for logging purposes?

Reply:
Yes, it only checks for the current version of node, as the `name` says :)
It may be unnecessary, but I copied-pasted and adapted from https://circleci.com/docs/2.0/circleci-images/#best-practices

Reply:
No harm in leaving it in! It will probably end up being useful in the future. :)